### PR TITLE
rand-extension example

### PR DESCRIPTION
**Pull request by ivan770**

### CODE REVIEW

**Review by coreggon11 — COMMENTED** [review]
lgtm, i just dont like formatting on some places, can we fix that? 😃

**Comment by ivan770** [comment]
> lgtm, i just dont like formatting on some places, can we fix that? smiley

Could you provide some places where fixes are required?

File: examples/rand-extension/chain-extension/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+#[cfg(feature = "substrate")]
+pub mod substrate;
+
+#[cfg(feature = "ink")]
+pub mod ink;
+
+#[obce::error]
+pub enum RandomReadErr {
+	FailGetRandomSource,
```
Comment (coreggon11) [inline examples/rand-extension/chain-extension/src/lib.rs]: spacing

File: examples/rand-extension/chain-extension/src/substrate.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+use obce::substrate::{
+	frame_support::traits::Randomness,
+	frame_system::Config as SysConfig,
+	pallet_contracts::{chain_extension::Ext, Config as ContractConfig},
```
Comment (coreggon11) [inline examples/rand-extension/chain-extension/src/substrate.rs]:
```suggestion
	pallet_contracts::{
	     chain_extension::Ext, 
	     Config as ContractConfig
	 },
```

**Review by coreggon11 — APPROVED** [review]
looking good

**Pull request merged**